### PR TITLE
Fabo/gov fixes

### LINKED ISSUE #1875: Optimistic updates: Proposal creation
We need to update the proposal list and deduct deposits.

### LINKED ISSUE #1876: Optimistic updates: Proposal deposits
We need to optimistically update the tokens after depositing on a proposal.

### PR DESCRIPTION
**Pull request by faboweb**
Closes #1875 
Closes #1876 

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- votes showing wrong amount @faboweb
+- depositing less then 1 atom not possible @faboweb
```
Comment (fedekunze) [inline CHANGELOG.md]: uh why not ?

Comment (faboweb) [reply CHANGELOG.md]: form validation prevented that, I fixed it

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,10 +10,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
 - [\#1306](https://github.com/cosmos/voyager/issues/1306) Withdraw fee distribution rewards @fedekunze
+- [\#1875](https://github.com/cosmos/voyager/issues/1875) optimistic updates for proposal creation @faboweb
+- [\#1876](https://github.com/cosmos/voyager/issues/1876) optimistic updates for depositing on proposals @faboweb
+
 
 ### Fixed
 
 - [\#2128](https://github.com/cosmos/voyager/pull/2128) remove loading status on validator page when disconnected from the network @fedekunze
+- [\#2167](https://github.com/cosmos/voyager/pull/2167) fixed votes showing wrong amount @faboweb
```
Comment (fedekunze) [inline CHANGELOG.md]:
```suggestion
- [\#2167](https://github.com/cosmos/voyager/pull/2167) fixed votes showing wrong tally @faboweb
```

**Comment by fedekunze** [comment]
coverage missing

**Comment by faboweb** [comment]
@fedekunze I think the case is:
- create proposal with less then needed deposit
- click on the proposal before the next block (optimistic entry into the proposals list)

**Comment by fedekunze** [comment]
> create proposal with less then needed deposit

That's not the case because the tx is submitted successfully

**Comment by faboweb** [comment]
hmm might have to do with the submissionError timeout. very strange. let's handle in another PR.

**Comment by fedekunze** [comment]
ok, let's create a tracking issue for it then

**Pull request merged**